Gameplay/entities/player/PlayerController.h

```diff
@@ -147,6 +147,7 @@ class PlayerController : public Script
 	SERIALIZE_FIELD(float, _dash_duration);
 	SERIALIZE_FIELD(float, _dash_distance);
 	SERIALIZE_FIELD(float, _dash_cooldown);
+	SERIALIZE_FIELD(int, _dash_scaler);
```

Comment:
```suggestion
	SERIALIZE_FIELD(unsigned int, _dash_scaler);
```
Hii! :3
I was testing the PR and when you add a negative number it brokes the position of the player game object and the camera look at too. I know it's not supposed to be negative in any case but I think we can change the _dash_scaler to unsigned int to be safe, or what do you think?
![image](https://user-images.githubusercontent.com/6739717/176324632-7eef93d5-a3da-4ccc-89d9-f15aa6415b82.png)